Sources/FSPageViewLayout.swift

```diff
@@ -142,8 +144,17 @@ class FSPagerViewLayout: UICollectionViewLayout {
         let attributes = FSPagerViewLayoutAttributes(forCellWith: indexPath)
         attributes.indexPath = indexPath
         let frame = self.frame(for: indexPath)
-        let center = CGPoint(x: frame.midX, y: frame.midY)
-        attributes.center = center
+        var midX: CGFloat = frame.midX
+        if horizontalAlignment == .left {
+            let offset = (collectionViewSize.width - frame.width)/2
+            let edgeSpacing = needsEdgeSpacing ? actualInteritemSpacing : 0
+            midX -= (offset - edgeSpacing)
+        } else if horizontalAlignment == .right {
+            let offset = (collectionViewSize.width - frame.width)/2
+            let edgeSpacing = needsEdgeSpacing ? actualInteritemSpacing : 0
+            midX += (offset - edgeSpacing)
```

Comment:
both left and right is the same for line 149 - 150
can u make it shorter again ?

Reply:
Yeah, of course.

Reply:
I try making it shorter but it just reduces one line of code which also reduces readability.
So, leaving the duplicated code in each condition is easier.